extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -730,6 +773,35 @@ export class AmpA4A extends AMP.BaseElement {
     });
   }
 
+  renderViaSafeFrame_() {
+    user().assert(this.creativeBody_,
+        'creativeBody missing in renderViaSafeFrame_?');
+    this.lifecycleReporter_.sendPing('renderCrossDomainStart');
+    utf8Decode(this.creativeBody_).then(creative => {
```

Comment:
Per your earlier suggestion, just passed `creativeBody` in.  Released it external to this method.